draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -631,7 +631,8 @@ DNS queries.
 HTTP/2 provides further advice about the use of compression
 ({{RFC7540}} Section 10.6) and padding ({{RFC7540}} Section 10.7 ).
 DoH Servers can also add DNS padding {{RFC7830}} if the DoH client requests
-it in the DNS query.
+it in the DNS query. Additionally, guidelines for choosing the padding length
+can be found in {{draft-ietf-dprive-padding-policy}}.
```

Comment:
that's a reference to experimental phrased as if its standards track guidance...

Reply:
I can modify it, and re-send a pull request. dprive-padding-policy has been implemented in open source software such as pydig & knot.

Reply:
Agreed.  I would perhaps say "initial attempts to offer guidance on choosing the padding length..."

Reply:
Thanks @moonshiner for the suggestion.